tools/ci_build/github/linux/docker/Dockerfile.ubuntu_openvino

```diff
@@ -2,33 +2,44 @@ ARG OS_VERSION=18.04
 FROM ubuntu:${OS_VERSION}
 
 ARG PYTHON_VERSION=3.5
-ARG OPENVINO_VERSION=2020.4
+ARG OPENVINO_VERSION=2021.1
 
 ADD scripts /tmp/scripts
 RUN /tmp/scripts/install_ubuntu.sh -p $PYTHON_VERSION -d EdgeDevice && \
     /tmp/scripts/install_deps.sh -p $PYTHON_VERSION -d EdgeDevice
 
 RUN apt update && apt install -y libnuma1 ocl-icd-libopencl1 && \
-    rm -rf /var/lib/apt/lists/*
-
-RUN /tmp/scripts/install_openvino.sh -o ${OPENVINO_VERSION} && \
-    rm -rf /tmp/scripts
+    rm -rf /var/lib/apt/lists/* /tmp/scripts
 
 WORKDIR /root
 
-ENV INTEL_OPENVINO_DIR /data/openvino/openvino_${OPENVINO_VERSION}.287
+ENV INTEL_OPENVINO_DIR /opt/intel/openvino_${OPENVINO_VERSION}.110
 ENV LD_LIBRARY_PATH $INTEL_OPENVINO_DIR/deployment_tools/inference_engine/lib/intel64:$INTEL_OPENVINO_DIR/deployment_tools/ngraph/lib:$INTEL_OPENVINO_DIR/deployment_tools/inference_engine/external/tbb/lib:/usr/local/openblas/lib:$LD_LIBRARY_PATH
 
 ENV PYTHONPATH $INTEL_OPENVINO_DIR/tools:$PYTHONPATH
 ENV IE_PLUGINS_PATH $INTEL_OPENVINO_DIR/deployment_tools/inference_engine/lib/intel64
-
-RUN wget https://github.com/intel/compute-runtime/releases/download/19.15.12831/intel-gmmlib_19.1.1_amd64.deb && \
-    wget https://github.com/intel/compute-runtime/releases/download/19.15.12831/intel-igc-core_1.0.2-1787_amd64.deb && \
-    wget https://github.com/intel/compute-runtime/releases/download/19.15.12831/intel-igc-opencl_1.0.2-1787_amd64.deb && \
-    wget https://github.com/intel/compute-runtime/releases/download/19.15.12831/intel-opencl_19.15.12831_amd64.deb && \
-    wget https://github.com/intel/compute-runtime/releases/download/19.15.12831/intel-ocloc_19.15.12831_amd64.deb && \
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN wget https://apt.repos.intel.com/openvino/2021/GPG-PUB-KEY-INTEL-OPENVINO-2021 && \
+    apt-key add GPG-PUB-KEY-INTEL-OPENVINO-2021 && rm GPG-PUB-KEY-INTEL-OPENVINO-2021 && \
+    cd /etc/apt/sources.list.d && \
+    echo "deb https://apt.repos.intel.com/openvino/2021 all main">intel-openvino-2021.list && \
```

Comment:
Wasn't '[trusted=yes]' needed in this line?

Reply:
That is required only if we have proxy.